install.sh

```diff
@@ -14,7 +14,7 @@ _QUESTION="[?]"
 _SUBITEM="\t$_ITEM"
 _SUBIMPORTANT="\t$_IMPORTANT"
 _SUBQUESTION="\t$_QUESTION"
-
+_UBUNTU_VERSION="$(lsb_release -r)"
```

Comment:
Looks like there is already a `$_OS_CODENAME` variable set elsewhere that accomplishes this same thing.  Could you switch the code to use that instead?